compiler/rustc_hir_typeck/src/method/suggest.rs

```diff
@@ -323,17 +323,50 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 }
             }
         }
+        //issue 108713 forgotten writer warning
+        let is_write_macro = sugg_span
+            .ctxt()
+            .outer_expn_data()
+            .macro_def_id
+            .map_or(false, |def_id| tcx.is_diagnostic_item(sym::write_macro, def_id));
```

Comment:
This should apply to `writeln!` too
```suggestion
            .map_or(false, |def_id| tcx.is_diagnostic_item(sym::write_macro, def_id) ||  tcx.is_diagnostic_item(sym::writeln_macro, def_id));
```

Reply:
Yep! Thanks 😃